pallets/file-system/src/utils.rs

```diff
@@ -271,9 +291,6 @@ where
             Error::<T>::BspAlreadyConfirmed
```

Comment:
I believe this is unecessary since it is impossible due to the check above. I can't use `expect_or_error` anyways, because it does not return the value of the execution. it is simply validating a condition is true.